CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 # Release Notes
 
+## 3.1.3
```

Comment:
This looks to be just slightly outdated, could you rebase from master?

Clearly we need to get this reviewed and merged quickly before anything else even small comes in :-)

Reply:
Right yes, I wasn't expecting a 3.1.4 😉